web-ui/src/main/resources/catalog/locales/en-v4.json

```diff
@@ -406,6 +406,7 @@
   "staticPageFormat-TEXT": "Plain text content",
   "staticPageStatus-HIDDEN": "Visible only to the administrator",
   "staticPageStatus-PRIVATE": "Visible to logged users",
+  "staticPageStatus-PRIVATE_GROUP": "Visible to logged users of certain group",
```

Comment:
"Visible to users belonging to the group",

Reply:
changed